src/cephadm/cephadm

```diff
@@ -3926,16 +3926,25 @@ def command_bootstrap(ctx):
         # first get latest mgrmap epoch from the mon.  try newer 'mgr
         # stat' command first, then fall back to 'mgr dump' if
         # necessary
-        try:
-            out = cli(['mgr', 'stat'])
```

Comment:
```suggestion
            out = retry(lambda: cli(['mgr', 'stat']))

Reply:
Thanks for the code review and please find this updated patch.

As per your suggestion, I used a new retry() function in order to minimize changes to the existing wait_for_mgr_restart() function. Because json.loads() is throwing the unhandled exception, not cli(), though I need to re-call cli() if a JSON exception is thrown, I had the retry() function call json.loads().